addons/player-thumb/userstyle.css

```
@@ -0,0 +1,10 @@
+.sa-project-thumb {
+  position: absolute;
+  top: 1px;
+  bottom: 1px;
+  left: 1px;
+  right: 1px;
+  width: calc(100% - 2px);
+  height: calc(100% - 2px);
+  border-radius: calc(0.5rem - 1px);
+}
```

The thumbnail is stretched while the loading screen is visible and it changes size when the project loads. This could be fixed by adding a "loading" class and some CSS:
```suggestion
}

.sa-project-thumb-loading {
  top: calc(2.75rem + 1px);
  height: calc(100% - 2.75rem - 2px);
}
```